api/request.go

```diff
@@ -19,6 +22,7 @@ type VolumeCreateRequest struct {
 	Type           string
 	IOPS           int64
 	PrepareForVM   bool
+	FSType         string
```

Comment:
device mapper has an another option for FS type. We don't need to update it to follow this for now, but if this option won't work on device mapper, it needs to be documented.